svc/AlertBannerService.ts

```diff
@@ -50,16 +50,18 @@ export class AlertBannerService extends HoistService {
         if (!this.enabled) return;
 
         const data: AlertBannerSpec = await XH.fetchJson({url: 'xh/alertBanner'}),
-            {active, expires, publishDate, message, intent, iconName, enableClose, clientApps} =
-                data,
+            {active, expires, publishDate, message, intent, iconName, enableClose, limitTo} = data,
             {lastDismissed, onClose} = this;
 
         if (
             !active ||
             !message ||
             (expires && expires < Date.now()) ||
             (lastDismissed && lastDismissed > publishDate) ||
-            !this.isTargetedApp(clientApps)
+            isEmpty(limitTo) ||
```

Comment:
This looks wrong - believe instead you want all the helper methods below to return true if no limits spec'd